src/main/java/blackjack/domain/user/Dealer.java

```diff
@@ -0,0 +1,19 @@
+package blackjack.domain.user;
+
+public class Dealer extends User {
+
+    public static final String DEALER = "딜러";
+    public static final int THRESHOLD = 16;
+
+    public Dealer() {
+        super(DEALER);
+    }
+
+    public String showInitialCardInfo() {
+        return name + CARD + cards.getCardInfo().get(0);
```

Comment:
값을 get으로 꺼내서 반환하지 말고 객체에게 메시지로 물어보는게 좋습니다
```suggestion
        return name + CARD + cards.firstCard();
```